Error in rule create_database:
    jobid: 0
    output: output/db_name.txt

RuleException:
AttributeError in line 15 of /home/ddhoogduin/programs/KeyGenes-dataprocessor-v0.2/Snakefile:
'NoneType' object has no attribute 'execute'
  File "/home/ddhoogduin/programs/KeyGenes-dataprocessor-v0.2/Snakefile", line 15, in __rule_create_database
  File "/home/ddhoogduin/programs/KeyGenes-dataprocessor-v0.2/Database.py", line 49, in add_db
  File "/usr/lib/python3.6/concurrent/futures/thread.py", line 56, in run
Exiting because a job execution failed. Look above for error message
